planning/work-items/other-documents-sync/arch.md

```diff
@@ -22,33 +22,41 @@ Continuation of https://github.com/w3c/wot-thing-description/issues/2120
 
-  - Category: A1 / A2 / A3 / B / C (example)
+  - A3 (add to [Behavioral Assertions](https://w3c.github.io/wot-thing-description/#behavior)?)
 
-- [ ] 2. There MAY be other representations of a Thing such as an HTML-based user interface, simply an image of the physical entity, or even non-Web representations in closed systems. ([arch-other-thing-representations](https://www.w3.org/TR/wot-architecture11/#arch-other-thing-representations))
+- [x] 2. There MAY be other representations of a Thing such as an HTML-based user interface, simply an image of the physical entity, or even non-Web representations in closed systems. ([arch-other-thing-representations](https://www.w3.org/TR/wot-architecture11/#arch-other-thing-representations))
+  - D
 
-- [ ] 3. To be considered a Thing, however, at least one TD representation MUST be available. ([arch-td-mandatory](https://www.w3.org/TR/wot-architecture11/#arch-td-mandatory))
+- [x] 3. To be considered a Thing, however, at least one TD representation MUST be available. ([arch-td-mandatory](https://www.w3.org/TR/wot-architecture11/#arch-td-mandatory))
+  - A3 (add to [Behavioral Assertions](https://w3c.github.io/wot-thing-description/#behavior)?)
 
 #### 6.1.2 Links
 
-- [ ] 4. WoT Thing Descriptions and WoT Thing Models MAY link to other Things, WoT Thing Models, and other resources on the Web to form a Web of Things. ([arch-td-linking](https://www.w3.org/TR/wot-architecture11/#arch-td-linking))
+- [x] 4. WoT Thing Descriptions and WoT Thing Models MAY link to other Things, WoT Thing Models, and other resources on the Web to form a Web of Things. ([arch-td-linking](https://www.w3.org/TR/wot-architecture11/#arch-td-linking))
+  - A1 ([5.3.4.1 Link](https://w3c.github.io/wot-thing-description/#link))
 
-- [ ] 5. Things MUST be hosted on networked system components with a software stack to realize interaction through a network-facing interface, the WoT Interface of a Thing. ([arch-networked](https://www.w3.org/TR/wot-architecture11/#arch-networked))
+- [x] 5. Things MUST be hosted on networked system components with a software stack to realize interaction through a network-facing interface, the WoT Interface of a Thing. ([arch-networked](https://www.w3.org/TR/wot-architecture11/#arch-networked))
+  - D
 
 ### 6.5 Interaction Model
 
-- [ ] 6. In addition to navigation affordances (i.e., Web links), Things MAY offer three other types of Interaction Affordances defined by this specification: Properties, Actions, and Events. ([arch-affordances](https://www.w3.org/TR/wot-architecture11/#arch-affordances))
+- [x] 6. In addition to navigation affordances (i.e., Web links), Things MAY offer three other types of Interaction Affordances defined by this specification: Properties, Actions, and Events. ([arch-affordances](https://www.w3.org/TR/wot-architecture11/#arch-affordances))
```

Comment:
Based on experience, I would flip it since people first think of property, action, event and then add links